docs/products/kafka/kafka-connect/howto/gcp-bigquery-sink.md

```diff
@@ -1,380 +1,255 @@
 ---
 title: Create a sink connector from Apache Kafka® to Google BigQuery
 ---
-
-The [Google BigQuery sink connector](https://github.com/confluentinc/kafka-connect-bigquery) enables you to move data from an Aiven for Apache Kafka® cluster to a set of Google BigQuery tables for further processing and analysis.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+Set up the [BigQuery sink connector](https://github.com/Aiven-Open/bigquery-connector-for-apache-kafka) to move data from Aiven for Apache Kafka® into BigQuery tables for analysis and storage.
+
+## Prerequisites
+
+- An Aiven for Apache Kafka service with
+  [Apache Kafka Connect enabled](/docs/products/kafka/kafka-connect/howto/enable-connect),
+  or a [dedicated Kafka Connect cluster](/docs/products/kafka/kafka-connect/get-started#apache_kafka_connect_dedicated_cluster)
+- A Google Cloud project with:
+  - A [service account and JSON service key](https://cloud.google.com/docs/authentication/client-libraries)
+  - BigQuery API enabled
+  - A [BigQuery dataset](https://cloud.google.com/bigquery/docs/datasets) to store data
+  - Dataset access granted to the service account
+
+Collect the following details:
+
+- `GOOGLE_CLOUD_PROJECT_NAME`: Target Google Cloud project name
+- `GOOGLE_CLOUD_SERVICE_KEY`: Google Cloud service account key in JSON format
+
+  :::warning
+  When adding the service key to the connector configuration, provide it as an escaped
+  string:
+  - Escape all `"` characters as `\"`
+  - Escape any `\n` in the `private_key` field as `\\n`
+
+  Example:
+  `{\"type\": \"service_account\",\"project_id\": \"XXXXXX\", ...}`
+  :::
+
+- `BIGQUERY_DATASET_NAME`: BigQuery dataset name
+- `TOPIC_LIST`: Comma-separated list of Kafka topics to sink
+- (Only if using Avro) Schema Registry connection details:
```

Comment:
We don't have a standard for this, but I wonder if we want to use the same format as we do for optional steps? Something to discuss sometime. 

```suggestion
- Only if using Avro: Schema Registry connection details:
```